src/client.js

```diff
@@ -15,16 +15,32 @@ class ProtocolClient {
     this.socket.on("open", callbacks.onOpen);
     this.socket.on("close", callbacks.onClose);
     this.socket.on("error", callbacks.onError);
-    this.socket.on("message", message => this.onMessage(message));
+    this.socket.on("message", (message) => this.onMessage(message));
   }
 
   close() {
     this.socket.close();
   }
 
+  async setAccessToken(accessToken) {
+    accessToken = accessToken || process.env.REPLAY_API_KEY;
```

Comment:
I realize that the browser and the driver all use `RECORD_REPLAY_` as the prefix for all environment variables. Wondering if we should do `RECORD_REPLAY_API_KEY` here.

Reply:
Good call! I'll update this